### PR TITLE
#52: implement the close method

### PR DESCRIPTION
**Pull request by AVM82**
connect to #52

### CODE REVIEW

File: src/main/java/juja/sqlcmd/DatabaseManager.java [hunk 1]
```diff
@@ -57,11 +57,13 @@ private int getTableRowsCount(String tableName) throws SQLException {
...
-    public void closeConnection() {
+    public void close() {
         try {
-            connection.close();
-        } catch (SQLException e) {
-            System.out.println(e.getMessage());
+            if(connection != null && !connection.isClosed()) {
+                connection.close();
+            }
+        } catch (SQLException | NullPointerException sqlException) {
```
Comment (Shtramak) [inline src/main/java/juja/sqlcmd/DatabaseManager.java]: Why you decide to handle SQLException here? You'll catch it, write "The connection is not established!" to System.err and what than? Also, handling of NPE is a bad practice (http://bit.ly/2z3Cwjc), becides after 'if(connection != null && !connection.isClosed())' you won't have NPE in any case

**Review by Shtramak — APPROVED** [review]

File: src/main/java/juja/sqlcmd/DatabaseManager.java (hunk 1 above)
Comment (AVM82) [reply src/main/java/juja/sqlcmd/DatabaseManager.java]: Of course you're right, I need to sleep (

File: src/test/java/juja/sqlcmd/DatabaseManagerTest.java [hunk 2]
```diff
@@ -62,7 +62,7 @@ public void testGetTablesNameWhenDbHasNotTables() throws SQLException, ClassNotF
         preparationEnvironment();
```
Comment (ivshapovalov) [inline src/test/java/juja/sqlcmd/DatabaseManagerTest.java:62]: Maybe verb instead of noun?

**Review by ivshapovalov — CHANGES_REQUESTED** [review]

File: src/test/java/juja/sqlcmd/DatabaseManagerTest.java (hunk 2 above)
Comment (AVM82) [reply src/test/java/juja/sqlcmd/DatabaseManagerTest.java:62]: At the next pull request i will removing this method

**Review by adaleorion — APPROVED** [review]

**Review by kortov — COMMENTED** [review]
Please change pr acc to Andrey and Ivan

**Comment by adaleorion** [comment]
@AVM82 do you have change for this pull request ?

**Review by ivshapovalov — APPROVED** [review]

**Review by kortov — APPROVED** [review]

**Pull request merged**